packages/cli/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../typescript/tsconfig.library.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```

Comment:
we should place this in `../typescript/tsconfig.library.json`.

So all of our packages share the same config

Reply:
It is already in that file but it is not helping: https://github.com/celo-org/celo-monorepo/blob/master/packages/typescript/tsconfig.library.json#L11

Reply:
that's awkward...

try removing it and run `tsc --showConfig` and see if it is there

Reply:
It is there :/

Reply:
I am getting 

```
 $ yarn run celocli account:new
yarn run v1.17.3
$ /Users/ashishb/celo/celo-monorepo5/node_modules/.bin/celocli account:new
/Users/ashishb/celo/celo-monorepo5/packages/contractkit/lib/base.js:19
exports.AllContracts = Object.keys(CeloContract).map(function (k) { return CeloContract[k]; }, exports.as, any), exports.as = CeloContract[];
                                                                                                                                           ^
SyntaxError: Unexpected token ]
    at Object.<anonymous> (~/celo/celo-monorepo5/packages/contractkit/src/index.ts:3:1)
    at Object.<anonymous> (~/celo/celo-monorepo5/packages/cli/src/base.ts:1:1)
    at Module.m._compile (~/celo/celo-monorepo5/node_modules/ts-node/src/index.ts:473:23)
    at Object.require.extensions.(anonymous function) [as .ts] (~/celo/celo-monorepo5/node_modules/ts-node/src/index.ts:476:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

So, I will have to debug that for testing further.